Application/Validation/NameValidator.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Core.Dto;
+using Core.Dto.Request;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Validation
+{
+    public class NameValidator : AbstractValidator<NameDto>
+    {
+        public NameValidator(GeoLocationValidator geoLocationValidator)
+        {
+            RuleFor(x => x.Name)
+            .NotEmpty().WithMessage("Name is required.");
+
+            RuleFor(x => x.Meaning)
+                .NotEmpty().WithMessage("Meaning is required.");
+
+            RuleForEach(x => x.Etymology)
+                .SetValidator(new EtymologyValidator());
+
+            RuleForEach(x => x.Videos)
+                .SetValidator(new EmbeddedVideoValidator());
+
+            RuleForEach(x => x.GeoLocation)
+                .SetValidator(geoLocationValidator);
+
+            RuleFor(x => x.FamousPeople)
+                .NotNull().WithMessage("FamousPeople is required")
+                .Must(BeAValidCommaSeparatedString).WithMessage("FamousPeople must be a valid comma-separated string.");
+
+            RuleFor(x => x.Syllables)
+                .NotNull().WithMessage("Syllables is required.")
+                .Must(BeAValidHyphenSeparatedString).WithMessage("Syllables must be a valid hyphen-separated string.");
+
+            RuleFor(x => x.SubmittedBy)
+                .NotEmpty().WithMessage("Submitted By is required.");
+        }
+
+        private bool BeAValidCommaSeparatedString(CommaSeparatedString commaSeparatedString)
+        {
+            if(commaSeparatedString == null)
+            {
+                return false;
+            }
+
```

Comment:
You can use the implicit conversion to convert any CharacterSeparatedString (Comma, Hyphen) object into a list of strings, like so. That way, you don't have to repeat the splitting logic here. Applies to the method below as well.

`List<string> items = hyphenSeparatedString;`